### PR TITLE
Update Task3.java

### PR DESCRIPTION
**Pull request by maydin61**
All seem to ok, Let"s move forward.

### CODE REVIEW

**Review by rokudaimme — APPROVED** [review]

**Review by nannatalia — APPROVED** [review]
i think its ok

File: src/Task3.java [hunk 1]
```diff
@@ -3,5 +3,8 @@ public class Task3 {
     public static void main(String[] args) {
 
         System.out.println("Third line of our code");
+        
+        System.out.println("When can we start to 4.th line code ");
+
```
Comment (maydin61) [inline src/Task3.java:8]: System.out.println("As soon as possible :)")

Comment (maydin61) [reply src/Task3.java:8]: :)

**Review by maydin61 — COMMENTED** [review]
it is OKAY.

**Pull request merged**